diff --git a/include/abicheck_lab/math.h b/include/abicheck_lab/math.h
index 99c1fe4..523d286 100644
--- a/include/abicheck_lab/math.h
+++ b/include/abicheck_lab/math.h
@@ -8,6 +8,7 @@ class Calculator {
   Calculator() = default;
   int add(int left, int right) const;
   int multiply(int left, int right) const;
+  int subtract(int left, int right) const;
 };
 
 int api_version();
diff --git a/src/math.cc b/src/math.cc
index ed53075..936557d 100644
--- a/src/math.cc
+++ b/src/math.cc
@@ -8,6 +8,7 @@ namespace abicheck_lab {
 
 int Calculator::add(int left, int right) const { return left + right; }
 int Calculator::multiply(int left, int right) const { return left * right; }
+int Calculator::subtract(int left, int right) const { return left - right; }
 int api_version() { return 1; }
 
 }  // namespace abicheck_lab
